frontend/website/src/components/icons/player/pause.svelte

```diff
@@ -0,0 +1,8 @@
+<script lang="ts">
+	export let size: number = 24;
```

Comment:
previously i did this with rem and fontsize, curious what your thoughts are about that approach? Is this better todo?

Reply:
The problem is, as far as I know, I can't apply any styles to the icon from outside of the icon component itself. Which means I would need a wrapper element to apply the styles to which seems unnecessary.